spec/integ/Ipv6Generator.spec.js

```diff
@@ -0,0 +1,145 @@
+/*
+ * Contains integration tests for all Startup-initiated events.
+ */
+
+const envBundle = require("../util/env-bundle");
+const { Ipv6Generator } = require("../../lib/irc/Ipv6Generator");
+const { IrcClientConfig } = require("../../lib/models/IrcClientConfig");
+const { IrcServer } = require("../../lib/irc/IrcServer");
+
+describe("Ipv6Generator", () => {
+    const {env, roomMapping, test} = envBundle();
+    let generator;
+    let store;
+    const server = new IrcServer("domain", IrcServer.DEFAULT_CONFIG, "example.com");
+
+    beforeEach(async () => {
+        await test.beforeEach(env);
+        env.ircMock._whenClient(roomMapping.server, roomMapping.botNick, "connect", (client, fn) => {
+            fn();
+        });
+        await test.initEnv(env);
+        store = env.ircBridge.getStore();
+        generator = new Ipv6Generator(store);
+    });
+
+    afterEach(async () => test.afterEach(env));
+
+
+    it("should generate an IPv6 address", async function() {
+        const address = await generator.generate(
+            '2001:0db8:85a3::', new IrcClientConfig('@foo:example.com', 'irc.example.com'), server
```

Comment:
This prefix shows up a lot in this file, how we about we extract it to a variable somewhere?